### PR TITLE
Fix react native linking

### PR DESCRIPTION
**Pull request by lukewalczak**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -169,6 +169,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-share')
```
Comment (tbergq) [inline android/app/build.gradle]: can you change to implementation, compile is deprecated

File: package.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
   },
   "dependencies": {
     "react": "^16.3.2",
-    "react-native": "^0.55.4"
+    "react-native": "^0.55.4",
+    "react-native-share": "^1.0.27"
```
Comment (mrtnzlml) [inline package.json]: @tbergq Wrong file but not sure where you need it exactly. Shared or MMB?

File: android/app/src/main/java/com/reactnativeapp/MainApplication.java [hunk 3]
```diff
@@ -34,6 +35,7 @@ public boolean getUseDeveloperSupport() {
         protected List<ReactPackage> getPackages() {
             return Arrays.asList(
                     new MainReactPackage(),
+            new RNSharePackage(),
```
Comment (tbergq) [inline android/app/src/main/java/com/reactnativeapp/MainApplication.java]: Please indent this to align with the rest, `OCD` tick

File: package.json (hunk 2 above)
Comment (tbergq) [reply package.json]: Right, for now it will be in MMB

Comment (tbergq) [reply package.json]: To be specific, this should be added to `app/MMB/package.json`

**Comment by tbergq** [comment]
LGDM, thanks 😄

**Pull request merged**